sys/linux/bpf.txt

```diff
@@ -400,19 +408,25 @@ bpf_obj_get {
 	file_flags	flags[bpf_open_flags, int32]
 }
 
+bpf_attach_targets [
+	zero	const[0, int32]
```

Comment:
Should this be 0, or -1 as invalid fd?
However, I think it may be better to mark both fd's below as `[opt]` instead.

Reply:
There are a couple of attach targets that ignore this outright, so either would work. Other entries default to 0, so I just used that.

I'll try the opt suggestion.